pnpm-lock.yaml

```diff
@@ -1,159 +1,2555 @@
-lockfileVersion: '6.0'
+lockfileVersion: '9.0'
```

Comment:
Could you discard changes to the lockfile?

Reply:
yep, doing it right away.